src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -652,4 +652,10 @@
     <value>⚠ You have unsaved changes.</value>
     <comment>{Locked="⚠"}</comment>
   </data>
+  <data name="Nav_ProfileAddNew.Content" xml:space="preserve">

```

Comment:
```suggestion
  <data name="Nav_AddNewProfile.Content" xml:space="preserve">
```
Same reasoning as before. Just trying to align this with #8089 to reduce the burden on the localization folks. (make sure you change the uuid in the xaml file)